components/service/glean/src/test/java/mozilla/components/service/glean/TestUtil.kt

```diff
@@ -0,0 +1,25 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean
+
+import org.json.JSONObject
+
+fun checkPingSchema(content: JSONObject) {
+    val proc = ProcessBuilder(
+        listOf("./build/bootstrap/Miniconda3/bin/python", "-m", "glean_parser", "check")
+    ).redirectOutput(ProcessBuilder.Redirect.INHERIT)
+        .redirectError(ProcessBuilder.Redirect.INHERIT)
+    val process = proc.start()
+
+    val stdinWriter = process.outputStream.bufferedWriter()
+    val jsonString = content.toString()
```

Comment:
If you put `val jsonString =`  on the line before `val stdinWriter` then you can do something like:

```
with (stdinWriter) {
  write(jsonString)
  newLine()
  flush()
  close()
}
```